orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -1414,7 +1429,10 @@ private StringBuffer processSettingsImpl(StringBuffer extraLines) throws IOExcep
             if (meekBridges)
                 extraLines.append("ClientTransportPlugin meek_lite socks5 127.0.0.1:" + IPtProxy.MeekSocksPort).append('\n');
 
-            if (bridgeList != null && bridgeList.length() > 5) //longer then 1 = some real values here
+            if (snowflakeBridges)
+                extraLines.append("ClientTransportPlugin snowflake socks5 127.0.0.1:" + IPtProxy.SnowflakeSocksPort).append('\n');
+
+            if (bridgeList != null && bridgeList.length() > 10) //longer then 1 = some real values here
```

Comment:
This comment should be updating. I suppose this works but i wish we had a better way of checking for this

Reply:
Yeah, it is really bad. I will take a shot now at improving the whole approach.